lib/publiccloud/utils.pm

```diff
@@ -74,6 +74,11 @@ sub is_ec2() {
     return is_publiccloud && check_var('PUBLIC_CLOUD_PROVIDER', 'EC2');
 }
 
+# Check if we are on an AWS test run and VM running on the host with ARM architecture
+sub is_ec2_arm() {
+  return is_ec2() && get_var('FLAVOR') =~ 'ARM';
```

Comment:
I don't like it, that we check the `FLAVOR` for an architecture. But I don't have a better idea... 
And conclusion is, using a function to wrap that check is mandatory.
So +1 from me :)

Reply:
not sure if `FLAVOR` or `PUBLIC_CLOUD_ARCH` is better here. 
also you can define this from the machine type. i think all the arm use `a1`